docs/content/en/docs/configuration/keys.md

```diff
@@ -910,6 +911,16 @@ HAProxy Ingress won't refuse to change the default naming if dynamic update is `
 
 ---
 
+## Backend server ID assignment
+
+| Configuration key          | Scope     | Default      | Since |
+|----------------------------|-----------|--------------|-------|
+| `assign-backend-server-id` | `Backend` | `false     ` | TODO  |
+
+When `true`, each backend server will receive an `id` in HAProxy config based on the Kubernetes UID of the pod backing it. When using a hash-based [`balance-algorithm`](#balance-algorithm) (for example `uri` or `source`) together with consistent hashing, this will maintain the stability of assignments when pods are added or removed — that is, a given URI component or source IP will mostly keep hashing to the same server. When this seetting is `false`, an addition or deletion in the server list may disturb the hash assignments of some or all of the remaining servers.
```

Comment:
please add also a few words about this config making dynamic update to not work - on a rolling update, the target IPs will change (which can be dynamically updated) but also the server id (which cannot).

Reply:
ps

- remove the trailing spaces in the `false` quote;
- since v0.13 - this feature will be merged.

Reply:
Okay. So I should recommend that anyone using this should disable `dynamic-scaling`, the same as with `backend-server-naming` above?

Reply:
Hmm I don't think so, only a note, a new phrase in an existent paragraph is enough, just to remind about this behavior. The backend naming is another matter due to the ability to wrongly name servers after a dynamic update.

Reply:
ps - with "not work" I mean a rolling update will make haproxy reload instead of dynamically update the endpoints.

Reply:
okay, so the user doesn't have to do anything, because the ingress recognizes that field as non-updatable... we just want to warn them. Correct?

Reply:
yep, a dynamic update only happens if the diff in the old and new state can be applied, a distinct PUID value is enough to go to the reload route.

Reply:
Okay, improved.